ddtrace/appsec/_asm_request_context.py

```diff
@@ -151,7 +151,7 @@ def finalise(self):
             callbacks = GLOBAL_CALLBACKS.get(_CONTEXT_CALL, []) if env.must_call_globals else []
             env.must_call_globals = False
             if env is not None and env.callbacks is not None and env.callbacks.get(_CONTEXT_CALL):
-                callbacks += env.callbacks.get(_CONTEXT_CALL)
+                callbacks = callbacks + env.callbacks.get(_CONTEXT_CALL)
```

Comment:
It's worth mentioning for future devs going through this code why `+=` isn't equivalent.

Reply:
Quoting [this SO answer](https://stackoverflow.com/questions/2347265/why-does-behave-unexpectedly-on-lists/2347423#2347423):
> For types that support both __iadd__ and __add__ you therefore have to be careful which one you use. a += b will call __iadd__ and mutate a, whereas a = a + b will create a new object and assign it to a. They are not the same operation!

Reply:
I also wonder if static analyzer could help warn here about an unintended side effect.